streampipes-client-e2e/go-client-e2e/go.sum

```diff
@@ -0,0 +1 @@
+github.com/apache/streampipes/streampipes-client-go v0.0.0-20240927072147-472569d5e1fb/go.mod h1:D7TeoMYCcSveKDVfJTkib3p1//1yA+egeNF6/j9vsuw=
```

Comment:
Maybe the sum file is not needed, the local code of streampipes-go-client is directly introduced in the mod file？